packages/deploy-tooling/src/base/config.ts

```diff
@@ -31,13 +31,17 @@ export function getConfigForLogging(
     }
 }
 
+export function loadEnv(): void {
```

Comment:
why do we wrap one function call into another function?

Reply:
I am also not sure why we need to call it twice in our coding then. Shouldn't there be one place, somewhere at the very beginning for the CLI use case where call config and then we are good?